packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -58,6 +59,7 @@ export function FormFieldWrapper({
         (placeholder && type === "select") ||
         // Naively assume that if the the type is tel, it is the InputPhoneNumber
         (placeholder && type === "tel"),
+      [styles.text]: type === "textarea" || type === "text",
```

Comment:
moved the label styles that now apply to both text and text area into a new style of `text`